includes/class-give-donor.php

```diff
@@ -790,6 +790,26 @@ public function get_notes_count() {
 	}
 
 	/**
+	 * Get the purchase value.
+	 *
+	 * @since 1.8.17
+	 *
+	 * @return string|float
+	 */
+	public function get_purchase_value() {
```

Comment:
@emgk Please rename this function to `get_total_donation_amount` and filter name to `give_get_total_donation_amount`

Reply:
Agreed. The name should be updated to how @ravinderk has mentioned.

Reply:
@DevinWalker I will update as @ravinderk mentioned, currently working on it.